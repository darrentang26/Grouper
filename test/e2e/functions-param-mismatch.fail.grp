let [Int] -> Int square = lambda (Int x)
    x * x end
in print (square "hi")