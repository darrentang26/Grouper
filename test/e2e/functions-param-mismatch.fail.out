Fatal error: exception Semant.Failure("expected parameter of type Int but recieved argument of type String")
